src/plugins/irc-events/error.js

```diff
@@ -44,15 +44,27 @@ module.exports = function(irc, network) {
 	});
 
 	irc.on("nick in use", function(data) {
+		let message = data.nick + ": " + (data.reason || "Nickname is already in use.");
+
+		if (irc.connection.registered === false) {
+			message += " We will try to set this nick back if we see old client quit.";
```

Comment:
Probably needs better wording (who is we?)

Reply:
Instead of "we", I would use "I". Some suggestions:
* `Nickname is already in use. Will retry nickchange when this nick is seen quitting.`
* `Nickname is already in use. I will retry nickchange when this nick is seen quitting.`
* `Nickname is already in use. Will retry when this nick is seen quitting.`
* `Nickname is already in use. I will retry when this nick is seen quitting.`
* `Nickname is already in use. Attempting to get it back...`

Reply:
I think "I" is no better, because the issue is we are there going "Who is I?". Whereas if we actually did "The Lounge will", or "TL will" if we insist that is too long, then it's very clear what is doing it. We don't have to figure out if it's the lounge, the server, or a bot that we haven't noticed.

Reply:
We already use wording like "and will not reconnect.", and I don't think we ever reference TL. With that in mind, perhaps this?

> Nickname is already in use. An attempt to use it will be made when this nick is seen quitting.

Reply:
Yeah I guess that could work. Certainly sounds less bad than "we" or "I"